addon/pods/components/rui-error-template/template.hbs

```diff
@@ -14,5 +14,14 @@
     <div class="rui-error-template-tagline">
       {{tagline}}
     </div>
+    {{#if link}}
+      <div class='rui-error-button'>
+        <a href={{link}}>
```

Comment:
This is not an optimal way of doing this. Links within the ember app (as in the new ember-participant login) will create a full refresh of the page which will create a bad user experience.

Potential options:

- Have it send an action that can be controlled on the app to go where it needs to go.
- Allow for sending of route instead of a link.  If its a route, the output would look like this:

```hbs
{{#if route}}
  <div class='rui-error-button'>
    {{#link-to route}}
      {{#rui-button size='lg'}}
        {{link-message}}
      {{/rui-button}}
    {{/link-to}}
  </div>
{{/if}}
```